src/Testing/src/AssemblyTestLog.cs

```diff
@@ -5,6 +5,7 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
```

Comment:
This is a fairly big change given this isn't a shipping package. I'm not even sure why it's packable. Might want to revert all changes under src/Testing. Up to you.

I should have realized this when I wrote up #24347.